sdk/daml-lf/language/src/main/scala/com/digitalasset/daml/lf/language/Ast.scala

```diff
@@ -1237,14 +1237,13 @@ object Ast {
       // in the context of upgrades. They will not be considered for upgrade checks.
       isUtilityPackage: Boolean,
   ) {
+    def supportsUpgrades(pkgId: Ref.PackageId) =
+      LanguageVersion.supportsPackageUpgrades(languageVersion) && (!UtilityDamlPrimPackages.check(
```

Comment:
not sure we need anymore ` LanguageVersion.supportsPackageUpgrades(languageVersion)` in `main`, everything is upgradeable in this branch.

Reply:
As long as we still have LF1.x versions in LanguageVersion I'd rather keep this check complete - when we remove those versions from LanguageVersion we can delete this expression.